### ISSUE TITLE
Human Pose estimation-001 model input size mismatch

### ISSUE THREAD

**Issue by Ai-econ** [openvinotoolkit/open_model_zoo#3908]
Model input size mentioned in accuracy-check yaml file  was not aligned with model input size mentioned in Readme.
What is actual model input size for the accuracy so that we able to achieve 42% map and is it AP 0.5 or AP 0.5-0.95

**Accuracy-check yaml**: [https://github.com/openvinotoolkit/open_model_zoo/blob/270dcb5d995b7b5f1db2c08c0db3283a78393e42/models/intel/human-pose-estimation-0001/accuracy-check.yml#L17]

**README**:[https://github.com/openvinotoolkit/open_model_zoo/blob/270dcb5d995b7b5f1db2c08c0db3283a78393e42/models/intel/human-pose-estimation-0001/README.md?plain=1#L28]

**Comment by Wovchena** [comment]
The size specified in accuracy-check.yml is actual model input size for the accuracy so that we able to achieve 42% map. README.md states the shape that you get after the model is converted. That's a common "technique" when shape for inference doesn't match the shape for accuracy.

**Comment by Ai-econ** [comment]
Thanks a lot for quick response

**Comment by eaidova** [comment]
@Ai-econ for achieving accurate results model required preserving aspect ratio of input image, during accruacy validation model input was reshaped according provided preprocessing rules

**Comment by Ai-econ** [comment]
@eaidova Where i can find preprocessing rules or guidelines so we can follow the same

 If i am not wrong,  model was trained with higher resolution which results in  good accuracy. However if we deploy the model with lower resolution just by converting model input shape will leads to decrease my accuracy and issue in extracting features right ?

**Comment by eaidova** [comment]
@Ai-econ they provided in accuracy checker config preprocessing section. Image resized with preserving aspect ratio, where height size will be 380 and width resized according these rules
https://github.com/openvinotoolkit/open_model_zoo/blob/270dcb5d995b7b5f1db2c08c0db3283a78393e42/tools/accuracy_checker/openvino/tools/accuracy_checker/preprocessor/resize.py#L33 and then if image height or width not divisible to 8, it will be additionally padded.

To sum up, resolution used for accuracy evaluation will depends from image size and it is bigger then provided by default (in model, default height - 256, while in accuracy - 386).

so resolution what used in accuracy is more then default im
